components/Calendar/index.js

```diff
@@ -1,29 +1,34 @@
 import { Calendar as MyCalendar, momentLocalizer } from "react-big-calendar";
 import "react-big-calendar/lib/css/react-big-calendar.css";
 import styled from "styled-components";
-import Link from "next/link";
 import { useCallback, useMemo, useState } from "react";
 import { Views } from "react-big-calendar";
 import moment from "moment";
```

Comment:
What are you using `moment` for? This is a very outdated library, they state themselves on their website that there are better alternatives nowadays, see https://momentjs.com/docs/#/-project-status/recommendations/
As you are using already other date libraries (and `moment` does not even show up as dependency in your `package.json` 🤔), I suggest to get remove it and use the date libraries you already rely on in your dependencies (`date-fns` etc.)